### ISSUE TITLE
[Config Support]: Configuring for Restreaming

### ISSUE THREAD

**Issue by dougransom** [blakeblackshear/frigate#8660]
### Describe the problem you are having

Have a doorbell cam, i want to restream it so I can view the video in VLC or so my neighbor can run his own NVR with it.

When I attempt to access what I think is the restreamed url, i get 404 errors:

`PS C:\Users\doug> ffprobe.exe -i rtsp://homeassistant.local:8554/h264Preview_01_main2
ffprobe version 6.0-full_build-www.gyan.dev Copyright (c) 2007-2023 the FFmpeg developers
  built with gcc 12.2.0 (Rev10, Built by MSYS2 project)
  configuration: --enable-gpl --enable-version3 --enable-shared --disable-w32threads --disable-autodetect --enable-fontconfig --enable-iconv --enable-gnutls --enable-libxml2 --enable-gmp --enable-bzlib --enable-lzma --enable-libsnappy --enable-zlib --enable-librist --enable-libsrt --enable-libssh --enable-libzmq --enable-avisynth --enable-libbluray --enable-libcaca --enable-sdl2 --enable-libaribb24 --enable-libdav1d --enable-libdavs2 --enable-libuavs3d --enable-libzvbi --enable-librav1e --enable-libsvtav1 --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs2 --enable-libxvid --enable-libaom --enable-libjxl --enable-libopenjpeg --enable-libvpx --enable-mediafoundation --enable-libass --enable-frei0r --enable-libfreetype --enable-libfribidi --enable-liblensfun --enable-libvidstab --enable-libvmaf --enable-libzimg --enable-amf --enable-cuda-llvm --enable-cuvid --enable-ffnvcodec --enable-nvdec --enable-nvenc --enable-d3d11va --enable-dxva2 --enable-libvpl --enable-libshaderc --enable-vulkan --enable-libplacebo --enable-opencl --enable-libcdio --enable-libgme --enable-libmodplug --enable-libopenmpt --enable-libopencore-amrwb --enable-libmp3lame --enable-libshine --enable-libtheora --enable-libtwolame --enable-libvo-amrwbenc --enable-libilbc --enable-libgsm --enable-libopencore-amrnb --enable-libopus --enable-libspeex --enable-libvorbis --enable-ladspa --enable-libbs2b --enable-libflite --enable-libmysofa --enable-librubberband --enable-libsoxr --enable-chromaprint
  libavutil      58.  2.100 / 58.  2.100
  libavcodec     60.  3.100 / 60.  3.100
  libavformat    60.  3.100 / 60.  3.100
  libavdevice    60.  1.100 / 60.  1.100
  libavfilter     9.  3.100 /  9.  3.100
  libswscale      7.  1.100 /  7.  1.100
  libswresample   4. 10.100 /  4. 10.100
  libpostproc    57.  1.100 / 57.  1.100
[rtsp @ 0000025bdd430540] method DESCRIBE failed: 404 Not Found
rtsp://homeassistant.local:8554/h264Preview_01_main2: Server returned 404 Not Found`

I have the RTSP restream port set to 8554 in the configuration UI for frigate.

Also, is there a way to run ffprobe on in HassOS?

### Version

Current version: 0.12.1

### Frigate config file

```yaml
# yaml-language-server: $schema=http://homeassistant.local:5000/api/config/schema.json
detectors:
  cpu1:
    type: cpu
mqtt:
  # Required host name
  host: homeassistant
  port: 1883
  #topic prefix must be  unique if running multiple instances
  topic_prefix: frigate
  stats_interval : 60
  user:  mqttuser
  password: hotdog
environment_vars:
  LIBVA_DRIVER_NAME  : i965

go2rtc:
  streams:
      doorbell_rec:  rtsp://doug:ransom@10.0.0.34:554/h264Preview_01_main
      doorbell_det:  rtsp://doug:ransom@10.0.0.34:554/h264Preview_01_sub



cameras:
  
  doorbell_det:
    ffmpeg:
      hwaccel_args: preset-intel-qsv-h264
      inputs:
        - path: rtsp://doug:ransom@10.0.0.34:554/h264Preview_01_sub
          roles: 
            - detect
  
  doorbell_rec:
    ffmpeg:
      hwaccel_args: preset-intel-qsv-h264
      inputs:
        - path: rtsp://doug:ransom@10.0.0.34:554/h264Preview_01_main
          roles: 
            - record

    detect:
      enabled: False # <---- disable detection until you have a working camera feed
      width: 640 # <---- update for your camera's resolution
      height: 480 # <---- update for your camera's resolution
      #- path: rtsp://user:pass10.0.0.87:554/h264Preview_01_sub 
    record:
      enabled : True
      

# Include all cameras by default in Birdseye view
birdseye:
  enabled: True
  mode: continuous
```

### Relevant log output

```shell
n/a
```

### Frigate stats

```json
n/a
```

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

CPU (no coral)

**Comment by NickM-27** [comment]
`rtsp://homeassistant.local:8554/h264Preview_01_main2`

This is incorrect, the url will be `rtsp://homeassistant.local:8554/stream` where stream is the name of the stream like doorbell_rec

**Comment by dougransom** [comment]
Thanks so much for the quick response.   I think i tried that before I realized I had to open the 8554 port in the addon configuration.   

I would suggest edits in this page:  [https://docs.frigate.video/guides/configuring_go2rtc]:
- the first step is to enable 8554 if you want restreaming outside the device.  easy to miss this
- a comment that the camera URL will rstp://localhost:8554/stream_name .  a new user like me has no idea what is going on if we are trying to learn frigate, ffmpeg, and gortc all at once.
- that the camera  URL in the cameras section be changed from the actual camera link to rtsp://localhost:8554/stream_name

Are there any ffmpeg things that should be copied from the cameras section to the gortc section?
